src/schemas/ev42/ev42_rw.cpp

```diff
@@ -42,35 +44,72 @@ uint64_t FlatbufferReader::timestamp(Msg const &msg) const {
 static FlatbufferReaderRegistry::Registrar<FlatbufferReader>
     RegisterReader("ev42");
 
-void HDFWriterModule::parse_config(rapidjson::Value const &config_stream,
-                                   rapidjson::Value const *config_module) {
-  if (auto x = get_int(&config_stream, "nexus.indices.index_every_kb")) {
-    index_every_bytes = uint64_t(x.v * 1024);
+void HDFWriterModule::parse_config(std::string const &ConfigurationStream,
+                                   std::string const &ConfigurationModule) {
+  auto ConfigurationStreamJson = json::parse(ConfigurationStream);
+  try {
+    index_every_bytes =
+        ConfigurationStreamJson["nexus"]["indices"]["index_every_kb"]
+            .get<uint64_t>() *
+        1024;
     LOG(Sev::Debug, "index_every_bytes: {}", index_every_bytes);
-  } else if (auto x = get_int(&config_stream, "nexus.indices.index_every_mb")) {
-    index_every_bytes = uint64_t(x.v * 1024 * 1024);
+  } catch (...) { /* it's ok if not found */
```

Comment:
Can't we catch anything more specific?

Reply:
Needs a bit more time, created ticket https://jira.esss.lu.se/browse/DM-925